features/machine/machine.feature

```diff
@@ -141,4 +141,39 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    And I wait for the steps to pass:
+    """
+    Given I clone a machineset named "machineset-24363"
+    """
+    Then I store the last provisioned machine in the :machine clipboard
+    And evaluation of `machine(cb.machine).node_name` is stored in the :noderef_name clipboard
+
+    Given I saved following keys to list in :taintsid clipboard:
+     | {"spec":{"taints": [{"effect": "NoExecute","key": "role","value": "master"}]}} | |
+     | {"spec":{"taints": [{"effect": "NoSchedule","key": "role","value": "master"}]}}| |
+
+    Then I repeat the following steps for each :id in cb.taintsid:
+    """
+    Given I run the :patch admin command with:
```

Comment:
could you use step `/^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$`?

Reply:
I am unable to figure out how to pass cb.machine in this .. I tired like below but it is not working as the step_definition is not supported for this ... let me know what shall I pass - 
   Given as admin I successfully merge patch resource machine "<%= cb.machine %>" with:
      | #{cb.id} |
      Then the step should succeed
      """
      Undefined dynamic step: "as admin I successfully merge patch resource machine "machineset-24363-lndbw" with:" (Cucumber::UndefinedDynamicStep)
      /verification-tests/verification-tests/features/step_definitions/meta_steps.rb:107:in `block (2 levels) in <top (required)>'

Reply:
Added above as you suggested , validated on local working fine -
        resourceVersion: ""
        selfLink: ""
      
      [11:44:32] INFO> Exit Status: 0
      [11:44:32] INFO> Shell Commands: rm -r -f -- /root/workdir/2aeba39a3db9-
      
      [11:44:32] INFO> Exit Status: 0
      [11:44:32] INFO> === End After Scenario: [MAO] Reconciling machine taints with nodes ===

1 scenario (1 passed)
12 steps (12 passed)
6m30.789s
[11:44:32] INFO> === At Exit ===